figure.html

```diff
@@ -325,14 +512,24 @@
 
 </li>
 
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
 <li>
 <p class="first">As the elements of the figure are saved as vector graphics, the PDF can be downloaded and further refined in your usual image editing app.</p>
```

Comment:
Maybe move this sentence to the end of this section so it's right above the Illustrator screenshot. Also I think it needs more emphasis that the screenshot has features added in Illustrator that are not supported in OMERO.figure. Otherwise people will be wondering how to do the arrows, ROIs etc.